### ISSUE TITLE
AssetPreload should lint only 1st party assets

### ISSUE THREAD

**Issue by nickbabcock** [Shopify/theme-check#693]
**Describe the bug**

The AssetPreload lint fires on the following line:

```html
<link rel="preload" as="script" href="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js">
```

If we take the suggestion and preload this 3rd party asset with the `preload_tag`

```liquid
{{ 'https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js' | preload_tag: as: 'script' }}
```

Shopify will return the following liquid error on page load:

```
Liquid error: Input must be a theme or shop asset URL. 
```

**Expected**

I expected that either the `preload_tag` would work with 3rd party assets or that the AssetPreload wouldn't give a false positive that leads to a liquid error.

**Additional context**
```
theme-check --version
1.12.1
```

**Comment by lukeh-shopify** [comment]
👋🏻 Hi @nickbabcock! Thanks for reporting this. Unfortunately the Ruby version of theme check has been deprecated for some time. Would you be able to confirm you are still having issues when using our [maintained version of theme check](https://shopify.dev/docs/storefronts/themes/tools/theme-check)? This is done using either the latest version of the Shopify CLI, or via the Shopify Liquid Visual Studio Code extension.

If you are still having issues, please raise an issue over in the [CLI](https://github.com/Shopify/cli) or [theme tools](https://github.com/Shopify/theme-tools) repositories. Thanks!
